extensions/wikia/PortableInfobox/tests/PortableInfoboxRenderServiceTest.php

```diff
@@ -25,8 +60,7 @@ private function setWikiaMobileSkin($bool) {
 	 * @dataProvider testRenderInfoboxDataProvider
 	 */
 	public function testRenderInfobox( $input, $expectedOutput, $description ) {
-		$this->setWikiaMobileSkin(false);
-
+		$this->infoboxRenderService = $this->getInfoboxRenderServiceMock( false, $input );
```

Comment:
It'll be safer to stop using $this-> here and instantiate $infoboxRenderService in the scope of each test